lib/zabbixapi/basic/basic_logic.rb

```diff
@@ -89,6 +89,16 @@ def get_full_data(data)
       )
     end
 
+    # DUPLICATE METHOD
+    # def get(data)
+    #   log "[DEBUG] Call get with parameters: #{data.inspect}"
+
+    #   @client.api_request(
+    #     method: "#{method_name}.get",
+    #     params: data
+    #   )
+    # end
+
```

Comment:
Is this still needed?

Reply:
Nope, the get method was abstracted in the alias class.